core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java

```diff
@@ -62,9 +62,10 @@ public BrewBuild findBrewBuildOfNVR(BrewNVR nvr) throws CausewayException {
             KojiSessionInfo session = login();
 
             KojiNVR knvr = new KojiNVR(nvr.getKojiName(), nvr.getVersion(), nvr.getRelease());
+            log.debug("Get build info of build {} from user '{}'.", knvr, session.getUserInfo().getUserName());
```

Comment:
We use only one user (newcastle) for communicating with Brew. If you want the user who inititated the push, you need to propagate it somehow from the rest endpoint.

Reply:
I think it's fine to log it as user newcastle? We'll also log it in the requestor (whether orch or rhpam) to reflect the full story.

@vibe13 On this note, I think the log should be info rather than debug.

I think in theory on Prod we'd like to run at the INFO level rather than debug, and this would disappear.

Reply:
@thescouser89 I have changed the log level to info